README.md

```
@@ -582,24 +586,30 @@ Thanks to  [@rolandomysqldba](https://dba.stackexchange.com/users/877/rolandomys
 
 Connection and Authentication
 
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)